documentation/development-docs/generate-format.ps1xml-file.md

```diff
@@ -0,0 +1,358 @@
+
+The cmdlet class specifies the ```PSAzureSubscription``` class as an output type with the **OutputType attribute**:
+
+```Cs
+namespace Microsoft.Azure.Commands.Profile
+{
+    [Cmdlet(VerbsCommon.Get, "AzureRmSubscription", DefaultParameterSetName = ListByIdInTenantParameterSet),
+        OutputType(typeof(PSAzureSubscription))]
+    public class GetAzureRMSubscriptionCommand : AzureRmLongRunningCmdlet
+    {
+        public const string ListByIdInTenantParameterSet = "ListByIdInTenant";
+        public const string ListByNameInTenantParameterSet = "ListByNameInTenant";
+
+// omitted for brevity the rest of the definition.
+```
+
+The [PSAzureSubscription](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Common/Commands.Common.Authentication.ResourceManager/Models/PSAzureSubscription.cs) class contains several public properties. 
+
+* Id
+* Name
+* State
+* SubscriptionId
+* TenantId
+* CurrentStorageAccountName
+* ExtendedProperties
+
+```Cs
+// code omitted for brevity 
+namespace Microsoft.Azure.Commands.Profile.Models
+{
+    public class PSAzureSubscription : IAzureSubscription
+    {
+
+// code omitted for brevity 
+
+        public string Id { get; set; }
+
+        public string Name { get; set; }
+
+        public string State { get; set; }
+
+        public string SubscriptionId { get { return Id; } }
+
+        public string TenantId
+        {
+            get
+            {
+                return this.GetTenant();
+            }
+            set
+            {
+                this.SetTenant(value);
+            }
+        }
+
+        public string CurrentStorageAccountName
+        {
+            get
+            {
+                return GetAccountName(CurrentStorageAccount);
+            }
+        }
+
+        public IDictionary<string, string> ExtendedProperties { get; }
+
+// code omitted for brevity 
+```
+
+PowerShell uses these properties for the cmdlet table formated output:
+
+```PowerShell
+PS C:\> Get-AzureRmSubscription | Format-Table
+
+Id                                   Name                      State   SubscriptionId                       TenantId                             CurrentStorageAcc
+                                                                                                                                                 ountName
+--                                   ----                      -----   --------------                       --------                             -----------------
+c9cbd920-c00c-427c-852b-c329e824c3a8 Azure SDK Powershell Test Enabled c9cbd920-c00c-427c-852b-c329e824c3a8 72f988bf-86f1-41af-91ab-7a64d1d63df5
+6b085460-5f21-477e-ba44-4cd9fbd030ef Azure SDK Infrastructure  Enabled 6b085460-5f21-477e-ba44-4cd9fbd030ef 72f988bf-86f1-41af-91ab-7a64d1d63df5
+
+
+```
+
+The default table output reveals some issues:
+1. **SubscriptionId** property values duplicates the **Id** property values, 
```

Comment:
I think it's important to call out these
- The selected fields don't feit in a standard window
- The columns are not displayed in oirder of importance to the customer for doign their work